### DISCUSSION TITLE
Concerning plugins

### DISCUSSION THREAD

**Discussion by LouisDiox** [Impostor/Impostor#232] [Q&A]
Hi !
Sorry if my question may sound stupid, but i don't understand what kind of file format are we supposed to add in the plugin folder ? Zip file ? Folder ? anything else ?
I checked the plugin example but i'm still a bit confused.

Thanks in advance

**Comment by gavinsteinhoff** [comment]
When you compile a plugin it will produce a .dll file. So that is what you will put in the plugin folder.
